p4-14/v1.0.3/tex/p4.tex

```diff
@@ -798,7 +798,7 @@
 
 A field instance is excluded from the calculation (i.e., it is treated as 
 if the instance is not listed in the input list) if the field's header 
-is not valid.
+is not valid.  (TBD: Is this true, or should the result of the calculation be undefined, since it will be fed an undefined value for that header field?)
```

Comment:
bmv2 skips invalid fields / headers in calculations, as this has proved very convenient in P4 programs which hash on L4 fields
this is also consistent with deparsing. IMO a field list calculation is just: 1) serializing the fields to a buffer (just like deparsing), and 2) applying a calculation over the buffer.